Sources/PickBetter/SegmentedBetterPickerStyle.swift

```diff
@@ -0,0 +1,170 @@
+// SegmentedBetterPickerStyle.swift
+// PickBetter
+//
+// Copyright © 2024 MFB Technologies, Inc. All rights reserved. All rights reserved.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+import Foundation
+import SwiftUI
+
+/// 'Segmented' style that mimics Apple's PickerStyle.segmented
+public struct SegmentedBetterPickerStyle: BetterPickerStyle {
+    /// Width of the entire Picker. Required property in order to calculate cell size correctly.
+    private var frameWidth: CGFloat
```

Comment:
Using `var` here isn't hurting anything but it would be nice if they were `let` instead